CITATION.cff

```diff
@@ -0,0 +1,166 @@
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "BIDS Stats Models Specification"
+
+version: 1.0.0-rc2
+
+abstract: BIDS Stats Models Specification.
+
+message: "If you are using the BIDS stats model, please cite it as below."
+
+repository-code: "https://github.com/bids-standard/stats-models.git"
+
+# identifiers:
+#   - description: This is the collection of archived snapshots of all releases of BIDSpm
+#     type: doi
+#     value: "10.5281/zenodo.3554331"
+
+contact:
+  - family-names: Markiewicz
+    given-names: Christopher J.
+    website: https://github.com/effigies
+    email: effigies@gmail.com
+  - family-names: de la Vega
+    given-names: Alejandro
+    website: https://github.com/adelavega
+
+authors:
+  # TEMPLATE
+  # - family-names: XXX
+  #   given-names: XXX
+  #   affiliation: XXX
+  #   website: XXX
+  #   email: XXX@XXX.XXX
+  #   orcid: "https://orcid.org/XXXX-XXXX-XXXX-XXXX"
+
+  #  Sorted alphabetically except for the 1rst and last authors
+
+  - family-names: Markiewicz
+    given-names: Christopher J.
+    website: https://github.com/effigies
+    email: markiewicz@stanford.edu
+    orcid: https://orcid.org/0000-0002-6533-164X
+    affiliation: Stanford University
+
+  - family-names: Bansal
+    given-names: Shashank
+    website: https://shashankbansal6.github.io/
+    email: shashankbansal56@gmail.com
+
+  - family-names: Blair
+    given-names: Ross W. 
+    website: https://github.com/rwblair
```

Comment:
@rwblair This will be used to credit you as a co-author of the BIDS Stats Models specification. Please confirm that you would like to be listed as an author and update the metadata as you see fit.